portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,33 +46,46 @@ private String convertToJsonUsingGson(List<String> messages) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get comments in datastore, by most recent order at the top.
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment").addSort("date", SortDirection.DESCENDING);
```

Comment:
would be great to add this as a class constant! That way in the future if you change this name, you'll only have to do it in one place :)

Reply:
Got it, done 👍